alica_engine/include/engine/allocationauthority/EntryPointRobotPair.h

```diff
@@ -38,6 +38,11 @@ struct hash<alica::EntryPointRobotPair>
     typedef alica::EntryPointRobotPair argument_type;
     typedef std::size_t value_type;
 
-    value_type operator()(argument_type& eprp) const { return std::hash<long int>()(eprp.getEntryPoint()->getId()) + eprp.getRobot()->hash(); }
+    value_type operator()(argument_type& eprp) const {
+        std::size_t seed = 0;
+        boost::hash_combine(seed, eprp.getEntryPoint()->getId());
```

Comment:
no need to introduce boost dependency here. I meant you can copy the code for hash_combine. But for now, we can just use + because that is how it was earlier